rdd/src/main/scala/ast/RDDMaskExpression.scala

```diff
@@ -0,0 +1,32 @@
+package maml.rdd.ast
+
+import maml.rdd.eval._
+import maml.ast._
+import maml.eval._
+
+import geotrellis.vector._
+import geotrellis.spark.mask._
+
+
+case class RDDMask(
```

Comment:
Expressions in MAML are supposed to be independent of the underlying representation. So its Mask on a raster rather than mask on RDD or Tile, where RDD/Tile are two possible instances of the raster.